mycli/packages/special/favoritequeries.py

```diff
@@ -57,5 +58,5 @@ def delete(self, name):
         self.config.write()
         return '%s: Deleted' % name
 
-from ...config import load_config
-favoritequeries = FavoriteQueries(load_config('~/.myclirc'))
+from ...config import read_config_file
+favoritequeries = FavoriteQueries(read_config_file(expanduser('~/.myclirc')))
```

Comment:
Can we add the `expanduser()` to the read_config_file() and read_config_files() as well? I think those two functions should be more defensive. 

Also, `expanduser()` is idempotent, so it should be harmless to call it on path values multiple times.